tests/pyconverter-test/baselines/destructuring.ts

```diff
@@ -0,0 +1,12 @@
+function initValue() {
+
+    let ___tempvar2 = [1, 0, 3]
```

Comment:
Any particular reason this starts with tempvar2 instead of tempvar0?

Reply:
our python compiler makes multiple passes and i'm not resetting the variable on each pass. no one should ever see this code, so i decided not to fix it.